docs/getting-started/release-notes/v2_9_0.md

```diff
@@ -18,35 +18,28 @@ Zowe Version 2.9.0 contains the enhancements that are described in the following
 * `zowe.certificates.pem` is no longer needed when using keyrings. ([#448](https://github.com/zowe/zlux-server-framework/pull/448))
 
 #### Zowe Common C
-* `configmgr's` zOS module now has a `resolveSymbol` function which takes a string starting with `&` which can be used to resolve static and dynamic ZOS symbols. ([#378](https://github.com/zowe/zowe-common-c/pull/378))
+* `configmgr's` zOS module now has a `resolveSymbol` function which takes a string starting with `&` which can be used to resolve static and dynamic z/OS symbols. ([#378](https://github.com/zowe/zowe-common-c/pull/378))
```

Comment:
Should "`configmgr's` zOS module" be changed to "`configmgr's` z/OS module" ?

Reply:
👆🏽 what Trae said 😋

Reply:
done.